admin/config/locales/en.yml

```diff
@@ -746,6 +746,21 @@ en:
           save_asset: Save Asset
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
+            <p>
+              <strong>Favicons:</strong> If you want to use an image asset as a favicon for the site,
+              you can add the <code>favicon</code> tag. Additionally, you can specify use for a
+              specific size favicon by tagging it with <code>favicon-HxW</code>. For example, a tag of
+              <code>favicon-32x32</code> will indicate the asset should be used for the favicon when
+              it's size is 32x32.
```

Comment:
Similar to explaining sharing below, should we briefly explain why you might use the different sizes?